src/PrestaShopBundle/Command/PrintCommandsAndQueriesForDocsCommand.php

```diff
@@ -0,0 +1,238 @@
+
+/**
+ * Prints all existing commands and queries to .md file for documentation
+ */
+class PrintCommandsAndQueriesForDocsCommand extends ContainerAwareCommand
+{
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'prestashop:print-docs:commands-and-queries';
+
+    /**
+     * Option name for providing destination directory path
+     */
+    private const DESTINATION_DIR_OPTION_NAME = 'dir';
+
+    /**
+     * Option name for forcing command (remove all confirmations)
+     */
+    private const FORCE_OPTION_NAME = 'force';
+
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    /**
+     * @var Environment
+     */
+    private $twigEnv;
+
+    /**
+     * @var StringModifierInterface
+     */
+    private $stringModifier;
+
+    /**
+     * @param Filesystem $filesystem
+     * @param Environment $twigEnv
+     * @param StringModifierInterface $stringModifier
+     */
+    public function __construct(
+        Filesystem $filesystem,
+        Environment $twigEnv,
+        StringModifierInterface $stringModifier
+    ) {
+        parent::__construct();
+        $this->filesystem = $filesystem;
+        $this->twigEnv = $twigEnv;
+        $this->stringModifier = $stringModifier;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure()
+    {
+        $this
+            ->setDescription(
+                'Prints available CQRS commands and queries to a file prepared for documentation' . PHP_EOL . PHP_EOL .
+                'Example: php ./bin/console prestashop:print-docs:commands-and-queries --dir=/path/to/doc_project/src/content/1.7/development/architecture/domain/references'
+            )
+            ->addOption(
+                self::DESTINATION_DIR_OPTION_NAME,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Path to file into which all commands and queries should be printed'
+            )
+            ->addOption(
+                self::FORCE_OPTION_NAME,
+                null,
+                InputOption::VALUE_NONE,
+                'Forces command to be executed without confirmations'
+            )
+        ;
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     *
+     * @return int|null
+     */
+    public function execute(InputInterface $input, OutputInterface $output): ?int
+    {
+        $destinationDir = $this->getDestinationDir($input);
+
+        if (!$this->confirmExistingFileWillBeLost($destinationDir, $input, $output)) {
+            $output->writeln('<comment>Cancelled</comment>');
+
+            return null;
+        }
+
+        $this->filesystem->remove($destinationDir);
+
+        $definitions = $this->getCommandHandlerDefinitions();
+        ksort($definitions);
+
+        foreach ($definitions as $domain => $definitionsByType) {
+            $content = $this->twigEnv->render('src/PrestaShopBundle/Command/views/cqrs-commands-list.md.twig', [
```

Comment:
You can't use `@PrestaShopBundle:blalbalbal`? Without it, modules are not ab le to plug themself on it

Reply:
Hmm, isnt that tag kinda attached to bundle/resoirces/views? 
I was ssuggested not to use this location for this particular template

Reply:
The view must be stored in `resources/views`

Reply:
That makes seense I will move the twig templates into `src/PrestaShopBundle/Resources/views/Commands`

Reply:
@PierreRambaud I just moved the templates, this allows to use the `@twig` service from framework So I also cleaned the extra services that were created